guides/common/modules/proc_importing-templates.adoc

```diff
@@ -79,3 +80,9 @@ For example `--filter '.*Ansible Default$'` imports various Ansible Default temp
 ----
 +
 If the import is successful, you receive `{"message":"Success"}`.
+
+[id="ansible_Importing_Templates_{context}"]
+.Ansible procedure
+* Use the `{ansible-namespace}.templates_import` module.
+
+For more information, see the Ansible Module documentation with `ansible-doc {ansible-namespace}.templates_import`.
```

Comment:
I'd go with lower case "modules". Uppercase is currently only in `guides/common/modules/proc_registering-a-host.adoc` once. I can create a follow-up PR to fix that afterwards.

Reply:
Fixed. And also: https://github.com/theforeman/foreman-documentation/pull/3434